buildSrc/subprojects/buildquality/src/main/kotlin/org/gradle/gradlebuild/buildquality/classycle/ClassyclePlugin.kt

```diff
@@ -35,7 +35,7 @@ open class ClassyclePlugin : Plugin<Project> {
     override fun apply(project: Project): Unit = project.run {
         val extension = extensions.create<ClassycleExtension>(classycleBaseName, project)
         configurations.create(classycleBaseName)
-        dependencies.add(classycleBaseName, "classycle:classycle:1.4@jar")
+        dependencies.add(classycleBaseName, "classycle:classycle:1.4.2@jar")
```

Comment:
`1.4.2` is necessary for Java 8 projects.

Reply:
I disagree, the build will fail if you use a version not compatible to Java 8, so why add a comment? Our whole pipeline is moving to Java 8 minimum anyway.